src/main/java/org/spongepowered/common/mixin/core/entity/MixinEntity.java

```diff
@@ -251,6 +253,29 @@ public void firePostConstructEvents() {
         this.isConstructing = false;
     }
 
+    @Inject(method = "addPassenger(Lnet/minecraft/entity/Entity;)V", at = @At(value="HEAD"), cancellable = true)
+    public void onAddPassenger(net.minecraft.entity.Entity passenger, CallbackInfo ci) {
+        if (!this.worldObj.isRemote) {
+            MountEntityEvent event = SpongeEventFactory.createMountEntityEvent(Cause.of(NamedCause.source(passenger)), this);
+            SpongeImpl.postEvent(event);
+            Thread.dumpStack();
```

Comment:
Yeah, I thought I caught everything, but I'm daft. Thanks for pointing that out.